content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -259,10 +307,11 @@ train:
   min_split: 2
 ```
 
-## Updating params and iterating
+## Modifying parameters
 
-We are definitely not happy with the AUC value we got so far! Let's edit the
-`params.yaml` file to use bigrams and increase the number of features:
+To work with parameters, let's assume (realistically) that we are not happy with
```

Comment:
`(realistically)` can we drop this? overall - can we err on keeping shorter sentences (fight for each word) + not changing if it's only about changing phrases, style, etc (low priority, makes it very hard to review etc)

Reply:
It was in the spirit of the original ("We are definitely not happy with") - but sure, dropped.

> can we err on keeping shorter sentences (fight for each word) + not changing if it's only about changing phrases, style, etc (low priority, makes it very hard to review etc)

I understand where this comes from of course, and am trying to keep it short, feel free to offer re-wording if you have some specific suggestions. but overall - giving the data which leads to this effort - let's please optimize for clarity and not **just** brevity 🙏